stablehlo/reference/Ops.cpp

```diff
@@ -165,35 +165,21 @@ SmallVector<Tensor> evalIfOp(const Tensor &pred, Region &trueBranch,
 
 Tensor evalIotaOp(int64_t iotaDimension, TensorType resultType) {
   Tensor result(resultType);
-  Type elType = result.getElementType();
+  Type elementTy = result.getElementType();
```

Comment:
Let's go even further and call this `elementType`? I don't think the `Ty` suffix is used anywhere in the file.